specs/cli/plugin-add.md

```diff
@@ -78,7 +78,7 @@ _The resulting `workspace.json` file will look like this:_
 }
 ```
 
-_The resulting `github-openai.json` file will look like this:_
+_The resulting OpenAI plugin named `ai-plugin.json` will look like this:_
```

Comment:
What is the impact of multiple plugins being used ai-plugin in the MOS manifest? I'm not 100% sure this is a required name.

Reply:
For OpenAI, it's a required name for sure. I doubled checked this with Rabia.
So far, even though the plugins node in MOS is an array, it's only possible to have 1 entry.